spec/lib/bamboozled/employee_spec.rb

```diff
@@ -100,6 +100,35 @@
     url.must_equal required_url
   end
 
+  it 'creates a new employee in BambooHR' do
```

Comment:
I would like to see this test cleaned up a bit. Using RSpec expect syntax instead. Move `xml_string`, `headers`, and `details` out of it (maybe to a fixture). That way it's easier to see the actual test. 

I don't know if it will hold up merging this PR but I wanted to document it :smiley: